tests/dotenv.net.Tests/DotEnv.Fluent.Tests.cs

```diff
@@ -127,6 +128,25 @@ public void ConfigShouldLoadEnvWithQuotedValues()
             .Be("single");
     }
 
+    [Fact]
+    public void ConfigLoadsMultilineEnvs()
+    {
+        DotEnv.Fluent()
+            .WithEnvFiles(MultiLinesEnvFileName)
+            .WithTrimValues()
+            .Load();
+
+        EnvReader.GetStringValue("DOUBLE_QUOTE")
+            .Should()
+            .Be("double");
+        EnvReader.GetStringValue("DOUBLE_QUOTE_MULTI_LINE")
+            .Should()
+            .Be("doubler");
```

Comment:
I think this should be 

```suggestion
            .Be("dou\nbler");
```